pkg/storage/store.go

```diff
@@ -55,6 +55,14 @@ type StoreCRC interface {
 	PutCRC(context.Context, string, io.Reader, bool, uint32) error
 }
 
+type StoreVersioned interface {
```

Comment:
`VersionedStore` ?

Reply:
otherwise, adding this new interface is the way to go

Reply:
> adding this new interface is the way to go

👍 ty for the sanity check